lib/Gemfile

```diff
@@ -45,6 +45,8 @@ gem "trollop",              "~>2.0",      :require => false
 gem "psych",                "~>2.0.12"
 gem "apipie-bindings",      "~>0.0.12",      :require => false
 
+gem 'byebug'
```

Comment:
yes the whole commit is to be removed, I just context switched, I didn't even wanted to push this commit here :-)